.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -79,6 +79,14 @@ class LoadTestsBuilder {
       return baseName
     }
   }
+
+  private static String getContextualDatasetKey(SDK sdk) {
+    String datasetKey = 'bigQueryDataset'
+    if (sdk == SDK.PYTHON) {
+      datasetKey = 'metrics_dataset'
```

Comment:
The one in Java SDK isn't good too, because in BigQuery performance tests there will be two parameters with the same name (one for metrics dataset and one for input/output dataset). I suggest the name `metrics_bq_dataset`. WDYT?